### PR TITLE
Ent - bug fixes

### PR DESCRIPTION
**Pull request by gayan-orangehrm**

### CODE REVIEW

**Review by lgtkaushalya — CHANGES_REQUESTED** [review]

File: components/src/core/components/Button/_mixins.scss [hunk 1]
```diff
@@ -107,6 +140,14 @@
   &:disabled {
     @include oxd-button-color-ghost(map-get($oxd-button, 'inactive'));
   }
+  &:hover {
```
[the remark this answers is not in the document]
Comment (gayan-orangehrm) [reply components/src/core/components/Button/_mixins.scss:143]: As per discussion with Mike,
- Main, secondary, "label button" buttons should have the same hover-over effect when on focus
- For ghost buttons it should be 5% darker in border and the text label in it

File: components/src/core/components/Button/button.scss [hunk 2]
```diff
@@ -128,11 +134,21 @@
   color: $oxd-interface-gray-darken-2-color;
   background-color: $oxd-background-white-color;
   cursor: pointer;
+  :deep(svg) {
```
[the remark this answers is not in the document]
Comment (gayan-orangehrm) [reply components/src/core/components/Button/button.scss:137]: We are not using glass button in the application at the moment

File: components/src/core/components/Calendar/calendar.scss [hunk 3]
```diff
@@ -112,8 +112,8 @@
     position: relative;
     list-style: none;
     display: flex;
-    margin: 0;
-    padding: 0;
+    margin: 0 !important;
```
[the remark this answers is not in the document]
Comment (gayan-orangehrm) [reply components/src/core/components/Calendar/calendar.scss:115]: fixed

File: components/src/core/components/Sheet/_variables.scss [hunk 4]
```diff
@@ -13,5 +13,5 @@ $oxd-sheet-white-shadow-color: $oxd-background-white-shadow-color !default;
 $oxd-sheet-white-color: $oxd-white-color !default;
 
 $oxd-sheet-border-radius: $oxd-border-radius !default;
-$oxd-sheet-padding-vertical: 1.5rem !default;
-$oxd-sheet-padding-horizontal: 1.5rem !default;
+$oxd-sheet-padding-vertical: 1.2rem !default;
```
[the remark this answers is not in the document]
Comment (gayan-orangehrm) [reply components/src/core/components/Sheet/_variables.scss:16]: No it wont. they've already overridden it with 1rem

File: components/src/core/components/FormDynamic/rules.ts [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    "url": (v: any) => (v && v.length <= 100) || 'Should be less than 100 characters',
+  },
+  {
+    "keywords": (v: any) => (v && v.length <= 254) || 'Should be less than 254 characters',
+  },
+  {
+    "email": (v: any) => !v || /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/.test(v) || 'E-mail must be valid',
+  },
+  {
+    "tel": (v: any) => !v || /^[0-9+()-\/]+$/.test(v) || 'Allows numbers and only + - / ( )',
```
[the remark this answers is not in the document]
Comment (gayan-orangehrm) [reply components/src/core/components/FormDynamic/rules.ts:19]: renamed

File: components/src/core/components/FormDynamic/rules.ts [hunk 6]
```diff
@@ -0,0 +1,21 @@
+/* eslint-disable */
+export const rulesList = [
+  {
+    "required": (v: any) => (!!v && v.trim() !== '') || 'Required',
+  },
+  {
+    "name": (v: any) => (v && v.length <= 30) || 'Should be less than 30 characters',
```
[the remark this answers is not in the document]
Comment (gayan-orangehrm) [reply components/src/core/components/FormDynamic/rules.ts:7]: fixed

File: components/src/core/components/Button/Button.vue [hunk 7]
```diff
@@ -18,7 +18,7 @@
       <oxd-icon
         v-if="iconRightName"
         :name="iconRightName"
-        class="oxd-button-icon"
+        class="oxd-button-icon oxd-button-icon-right"
```
[the remark this answers is not in the document]
Comment (gayan-orangehrm) [reply components/src/core/components/Button/Button.vue:21]: fixed

File: components/src/core/components/Button/_mixins.scss [hunk 8]
```diff
@@ -1,6 +1,11 @@
 @mixin oxd-button-color($background-color) {
   color: $oxd-button-font-color;
   background-color: $background-color;
+  :deep(svg) {
```
[the remark this answers is not in the document]
Comment (gayan-orangehrm) [reply components/src/core/components/Button/_mixins.scss]: He has provided a fix offline

**Pull request merged**